lua/terrortown/menus/gamemode/appearance/interface.lua

```diff
@@ -30,6 +32,17 @@ function CLGAMEMODESUBMENU:Populate(parent)
         convar = "ttt2_hud_enable_description",
     })
 
+    form:MakeComboBox({
+        label = "label_distance_unit",
+        convar = "ttt2_distance_unit",
+        choices = {
+            { title = TryT("choice_distance_unit_0"), value = 0 },
+            { title = TryT("choice_distance_unit_1"), value = 1 },
+            { title = TryT("choice_distance_unit_2"), value = 2 },
+            { title = TryT("choice_distance_unit_3"), value = 3 },
```

Comment:
These translation names should probably be named with the units to make it easier to verify that the correspondence is correct.

Reply:
good idea, though all the other choice\_\* entries in the translation files are like this, so i just did this for consistency